src/systems/user_commands/UserCommands.cc

```diff
@@ -695,24 +703,83 @@ bool PoseCommand::Execute()
     return false;
   }
 
+  const auto *poseCmdComp =
+    this->UpdateEntityPoseCommand(entity, msgs::Convert(*poseMsg));
+
+  // Also need to update descendants of the entity
+  auto *currentPose =
+    this->iface->ecm->Component<components::WorldPose>(entity);
+
+  const auto poseUpdateTransform = (currentPose) ?
+    currentPose->Data().Inverse() * poseCmdComp->Data() : poseCmdComp->Data();
+
+  std::unordered_map<Entity, math::Pose3d> cachedPoses;
+  for (const auto child : this->iface->ecm->Descendants(entity))
```

Comment:
Ideally the `FreeGroup` feature in ign-physics would handle updating the poses of nested models when a top level model's pose is changed in the same way it now handles the poses of disconnected links when a model's pose is changed. Is that something we can explore?

Reply:
I wasn't aware of that feature. But in this instance, it was an issue even for a model where links were connected by joints and I had to descend recursively. So it might work, but it would be used for more than just a free group so to speak.

Where would I test grouping elements in a nested model together into a free group?